fil_actors_shared/src/v8/builtin/singletons.rs

```diff
@@ -3,21 +3,28 @@
 
 use fvm_shared::address::Address;
 use fvm_shared::ActorID;
-use lazy_static::lazy_static;
+use paste::paste;
```

Comment:
I'm not sure it's adding a macro here. The code without it would give less mental overhead and wouldn't be that much longer. Why do you think macro is necessary?

Reply:
It's not, we have the initiation repetitive for 8 actors so i used macro thinking it will be easier to manage but since this list will not change anymore for v8 using macro here is not necessary, I'll keep it simple and remove this macro rule